.github/workflows/ci_cd.yml

```diff
@@ -1,7 +1,27 @@
 name: GitHub CI
+
 on:
   pull_request:
+  schedule:
+    - cron: '00 22 * * *'  # UTC time, may start 5-15 mins later than scheduled time
   workflow_dispatch:
+    inputs:
+      default-revn:
+        type: choice
+        description: 'Default revision number for containers.'
+        options: 
+        - '241'
+        - '232'
+        - '231'
+        default: '231'  # ensure this is the same as env.DEFAULT_REVN
```

Comment:
@RobPasMue @klmcadams what do you think about running all versions?  Now that I think about it, we want to make sure PyMechanical version X supports all mechanical versions before it was released. I'm not sure if we do that level of testing for other pyansys libraries.

Reply:
Some libraries do perform this kind of testing. I'd suggest that if possible, you do test against all of them

Reply:
@klmcadams @RobPasMue let's handle that in a separate issue without delaying this PR. For me right now the priority is to make sure 241 is covered before development of the backend closes